src/components/ExportButton/exportUtils.js

```diff
@@ -243,16 +245,16 @@ export const exportPdf = async (
   const ctx = canvas.getContext('2d');
 
   // Apply SVG overlay if provided
-  if (topic.exportConfig && topic.exportConfig.overlayImageUrl) {
-    const { overlayImageUrl, dateDe, dateFr, publisher, publishedAt, year } =
-      topic.exportConfig;
+  if (exportConfig && overlayImageUrl) {
+    const { dateDe, dateFr, publisher, publishedAt, year } = exportConfig;
```

Comment:
not generic enough if we have a new topic , we don't want to always add new values to replace.

in exportConfig create  a  getTemplateValues function that returns an  object : 
getTemplateValues() {
   return {
       year: '2023',
       publishedAt: '2023,    
       dateFr: '...'
       dateDe: '...
}

and then in exportUtils you just go throught these values are replace them in the svg
}

Reply:
done